drivers/windows/file_access_windows.cpp

```diff
@@ -91,16 +91,23 @@ Error FileAccessWindows::open_internal(const String &p_path, int p_mode_flags) {
 	path_src = p_path;
 	path = fix_path(p_path);
 
-	const WCHAR *mode_string;
+	String mode_string = L"";
```

Comment:
Use `"..."` or `U"..."`, L prefix will cause unnecessary conversion from UTF-16 (wchar_t on Windows) to UTF-32 (Godot string).

Reply:
Strings should now have the U prefix to prevent wide char resize for Windows.